### ISSUE TITLE
Some OSS libraries not present in corefx-progress

### ISSUE THREAD

**Issue by Maxwe11** [dotnet/corefx-progress#12]
- [ ] System.IO.Packaging
- [ ] System.Composition

**Comment by akoeplinger** [comment]
Yeah, I still hope they can run the update script more often: https://github.com/dotnet/corefx-progress/issues/9

**Comment by Maxwe11** [comment]
@akoeplinger these packages initially missing from lib-full & src-full

**Comment by akoeplinger** [comment]
Ah, good point!
